game-of-life-multiplatform/README.md

```diff
@@ -36,3 +36,12 @@ Run with `./gradlew desktop-jvm:run`
 
 ![gol](https://user-images.githubusercontent.com/517070/35706797-723ef2dc-07a7-11e8-857d-d4fc86a2e200.png)
 
+# iOS
+iOS client
+
+Requires xcode 9.2
```

Comment:
Where does this requirement come from?

Reply:
From [kotlin native](https://github.com/JetBrains/kotlin-native/blob/master/README.md)

Reply:
on macOS install Xcode 9.2